src/lib/ClaimCodeManager.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import crypto from 'crypto';
+
+type ClaimCodeObject = {
+  claimCode: string,
+  userID: string,
+}
+
+const CLAIM_CODE_LENGTH = 63;
+const CLAIM_CODE_LIVE_TIME = 5000 * 60; // 5 min
```

Comment:
//cc @durielz , what do you think about this pattern instead store claimCodes in User model.. and also have you some thoughts about appropriate CLAIM_CODE_LIVE_TIME?